package.json

```diff
@@ -7,13 +7,13 @@
     "history": "4.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "3.2.0"
```

Comment:
can we upgrade the `"@twilio/flex-ui` version as well as it's currently pointing to a `beta` version?

Reply:
it does resolve to 2.11 at the moment but happy to do it yes